onnx/onnx.proto

```diff
@@ -404,6 +466,22 @@ message TensorShapeProto {
   repeated Dimension dim = 1;
 }
 
+// Quantization parameters
+message QuantizationProto {
+  // This field MUST be present if and only if data type is QUINT8 or QUINT16.
+  // Zero point quantization parameter for quantized tensors. See description
+  // of quantized types for interpretation of this parameter.
+  // The value of this field must be in [0, 255] range for QUINT8 type, and
+  // within [0, 65535] range for QUINT16 type.
+  optional int32 zero_point = 2;
+  // This field MUST be present if and only if data type is QUINT8, QUINT16,
+  // QINT8, QINT16, or QINT32.
+  // Scale quantization parameter for quantized tensors. See description of
+  // quantized types for interpretation of this parameter.
+  // The value of this field must be in [2**14 ~= 6.10352e−5, 256] range.
+  optional float scale = 3;
+}
```

Comment:
@Maratyszcza Besides the scalar type "scale" and "zero_point", shall we also support the finer-grained quantization, e.g. channel-wise quantization? Is "TensorProto" a more general type for "scale" and "zero_point"? We also need to consider the axis along which to specify axis-wise "scale" and "zero_point". What do you think?